bootstrap/js/Axios.js

```diff
@@ -0,0 +1,89 @@
+//Add an event listener for the form submission to create a new user
+
+const form = document.querySelector("form");
+
+
+const formEvent = form.addEventListener('submit', event => {
+   event.preventDefault(); 
+   const username = document.querySelector("#username").value;
+   const email = document.querySelector("#email").value;
+   const message = document.querySelector("#message").value;
+
+   const user = { username, email, message };
+
+
+   createUser(user);
+});
+
+
+
+const createUser = (user) => {
+    let postObject = {...user}
+axios({
+    method: 'post',
+    url: ' https://gg-web-api.herokuapp.com/users/',
+    data: {
+      "username": `${postObject.username}`,
+      "email": `${postObject.email}`,
+      "message": `${postObject.message}`
+    }
+  });
+
+}
+
+createUser();
+
+//flash message when a user post data to the endpoint 
+(function($) {
+  $.fn.flash_message = function(options) {
+
+    options = $.extend({
+      text: 'Done',
+      time: 3000,
+      how: 'before',
+      class_name: ''
+    }, options);
+
+    return $(this).each(function() {
+      if ($(this).parent().find('.flash_message').get(0))
+        return;
+
+      var message = $('<span />', {
+        'class': 'flash_message ' + options.class_name,
+        text: options.text
+      }).hide().fadeIn('fast');
+
+      $(this)[options.how](message);
+
+      message.delay(options.time).fadeOut('normal', function() {
+        $(this).remove();
+      });
+
+    });
+  };
+})(jQuery);
+
+$('.button').click(function() {
+
+  $('#status-area').flash_message({
+    text: 'Thanks for filling out our form!, You can click on clear form button',
```

Comment:
Ingrid might have an issue with the way this message is phrased. I'd suggest that you think of something more user friendly.

Reply:
thanks for the feedback